tests/v2/validation/rbac/clusterandprojectroles/restrictedadmin_role_test.go

```diff
@@ -0,0 +1,117 @@
+	ra.session.Cleanup()
+}
+
+func (ra *RestrictedAdminTestSuite) SetupSuite() {
+	ra.session = session.NewSession()
+
+	client, err := rancher.NewClient("", ra.session)
+	require.NoError(ra.T(), err)
+
+	ra.client = client
+
+	log.Info("Getting cluster name from the config file and append cluster details in pr")
+	clusterName := client.RancherConfig.ClusterName
+	require.NotEmptyf(ra.T(), clusterName, "Cluster name to install should be set")
+	clusterID, err := clusters.GetClusterIDByName(ra.client, clusterName)
+	require.NoError(ra.T(), err, "Error getting cluster ID")
+	ra.cluster, err = ra.client.Management.Cluster.ByID(clusterID)
+	require.NoError(ra.T(), err)
+}
+
+func (ra *RestrictedAdminTestSuite) TestRestrictedAdminCreateCluster() {
+	subSession := ra.session.NewSession()
+	defer subSession.Cleanup()
+
+	_, restrictedAdminClient, err := rbac.SetupUser(ra.client, rbac.RestrictedAdmin.String())
+	require.NoError(ra.T(), err)
+	ra.T().Logf("Validating restricted admin can create an RKE1 cluster")
+	userConfig := new(provisioninginput.Config)
+	config.LoadConfig(provisioninginput.ConfigurationFileKey, userConfig)
+	nodeProviders := userConfig.NodeProviders[0]
+	nodeAndRoles := []provisioninginput.NodePools{
+		provisioninginput.AllRolesNodePool,
+	}
+	externalNodeProvider := provisioning.ExternalNodeProviderSetup(nodeProviders)
+	clusterConfig := clusters.ConvertConfigToClusterConfig(userConfig)
+	clusterConfig.NodePools = nodeAndRoles
+	kubernetesVersion, err := kubernetesversions.Default(restrictedAdminClient, clusters.RKE1ClusterType.String(), []string{})
+	require.NoError(ra.T(), err)
+
+	clusterConfig.KubernetesVersion = kubernetesVersion[0]
+	clusterConfig.CNI = userConfig.CNIs[0]
+	clusterObject, _, err := provisioning.CreateProvisioningRKE1CustomCluster(restrictedAdminClient, &externalNodeProvider, clusterConfig)
+	require.NoError(ra.T(), err)
+	provisioning.VerifyRKE1Cluster(ra.T(), restrictedAdminClient, clusterConfig, clusterObject)
+}
+
+func (ra *RestrictedAdminTestSuite) TestRestrictedAdminGlobalSettings() {
+
+	subSession := ra.session.NewSession()
+	defer subSession.Cleanup()
+
+	_, restrictedAdminClient, err := rbac.SetupUser(ra.client, rbac.RestrictedAdmin.String())
+	require.NoError(ra.T(), err)
+	ra.T().Log("Validating restricted Admin can list global settings")
+	steveRestrictedAdminclient := restrictedAdminClient.Steve
+	steveAdminClient := ra.client.Steve
+
+	adminListSettings, err := steveAdminClient.SteveType(settings.ManagementSetting).List(nil)
+	require.NoError(ra.T(), err)
+	adminSettings := adminListSettings.Names()
+
+	resAdminListSettings, err := steveRestrictedAdminclient.SteveType(settings.ManagementSetting).List(nil)
+	require.NoError(ra.T(), err)
+	resAdminSettings := resAdminListSettings.Names()
+
+	assert.Equal(ra.T(), len(adminSettings), len(resAdminSettings))
+	assert.Equal(ra.T(), adminSettings, resAdminSettings)
+}
+
+func (ra *RestrictedAdminTestSuite) TestRestrictedAdminUpdateGlobalSettings() {
```

Comment:
can you signify in the test name that this is a negative test? because this test is expecting an error.